### PR TITLE
implement loyalty points

### PR DESCRIPTION
**Pull request by limli**

### CODE REVIEW

**Comment by hiyingnn** [comment]
Looks good. Remember to update user guide and tests.

File: src/main/java/seedu/address/model/person/LoyaltyPoints.java [hunk 1]
```diff
@@ -26,6 +26,26 @@ public LoyaltyPoints(int points) {
...
+     */
+    public static boolean isValidLoyaltyPoints(int points) {
+        return 0 <= points; //TODO might want to set upper bound
+    }
+
+    /**
+     * Returns true if a given string is a valid loyalty points.
+     */
+    public static boolean isValidLoyaltyPoints(String points) {
+        int integerPoints = 0;
```
Comment (lingyanhao) [inline src/main/java/seedu/address/model/person/LoyaltyPoints.java]: why you initialise the value?

Comment (limli) [reply src/main/java/seedu/address/model/person/LoyaltyPoints.java]: because java warns me if i dont initialise it and try to use it :(

Comment (lingyanhao) [reply src/main/java/seedu/address/model/person/LoyaltyPoints.java]: can probably reduce the variable scope, something like
try {
    integerPoints = Integer.parseInt(points);
    return integerPoints >= 0;
} catch (....)

not important though, its just not very good practice to initialize a value to not use it

**Comment by zj-cs2103** [comment]
Reopened at https://github.com/cs2103-ay1819S2-w11-2/main/pull/75

**Pull request closed without merging**